src/test/scala/jsy/student/Lab2Spec.scala

```diff
@@ -250,8 +251,4 @@ class Lab2SpecRunner extends Lab2Spec(Lab2)
 // The test expects a corresponding .ans file with the expected result.
 class Lab2JsyTests extends JavascriptyTester(None, "lab2", Lab2)
 
-class Lab2Suite extends Suites(
```

Comment:
Just wondering why you're removing the Suite?

Reply:
The Suites object seems to be the root of the tests running twice issue, removing it causes 36 tests to run instead of 72 (there are only 32 tests in the repository).

Reply:
I see now.